client/src/main/java/io/pravega/client/stream/impl/ReaderGroupState.java

```diff
@@ -114,9 +114,9 @@ int getNumberOfReaders() {
      */
     @Synchronized
     int getRanking(String reader) {
-        List<String> sorted = distanceToTail.entrySet()
+        List<String> sorted = getRelativeSizes().entrySet()
```

Comment:
getRelativeSizes() returns a size for a reader using the computation.
The size is computed by the formula : Number of assigned segments * lag for the reader / max lag of all readers. This value can be > 1.0 right?

Reply:
Yes. The idea is that 1.0 means you have one segment and the one with the largest lag. So 2.0 means the reader has twice that much data to read.